3-UsingAPIs/Week2/assignment/ex1-programmerFun/index.js

```diff
@@ -16,29 +16,39 @@ Full description at: https://github.com/HackYourFuture/Assignments/blob/main/3-U
    url with `.shx`. There is no server at the modified url, therefore this 
    should result in a network (DNS) error.
 ------------------------------------------------------------------------------*/
-function requestData(url) {
-  // TODO return a promise using `fetch()`
+async function requestData(url) {
+
+  const response = await fetch(url);
+  
+  if (!response.ok) {
+    throw new Error("some error with API" +  response.status);
+  }
+  
+  return await response.json()
 }
 
 function renderImage(data) {
-  // TODO render the image to the DOM
-  console.log(data);
+  const {img, alt} = data;
+  const renderImg = document.createElement('img');
+
+  renderImg.src = img;
+  renderImg.alt = "picture with a fanfact";
```

Comment:
Why did you change this to an unhelpful description? The way it was previously was perfect:

```suggestion
  renderImg.alt = alt;
```